test/general/TestFill.py

```diff
@@ -96,12 +96,14 @@ def generate_player_data(multi_world: MultiWorld, player_id: int, location_count
     return PlayerDefinition(multi_world, player_id, menu, locations, prog_items, basic_items)
 
 
-def generate_locations(count: int, player_id: int, address: int = None, region: Region = None, tag: str = "") -> List[Location]:
+def generate_locations(multi_world: MultiWorld, count: int, player_id: int, address: int = None, region: Region = None,
+                       tag: str = "") -> List[Location]:
     locations = []
     prefix = "player" + str(player_id) + tag + "_location"
     for i in range(count):
         name = prefix + str(i)
         location = Location(player_id, name, address, region)
+        location.world = multi_world
```

Comment:
Why would you need `location.world` now? it seems you are using `location.parent_region.world` instead

Reply:
I was testing storing world on location.world and what the performance impact might be, apparently left that line in.